docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,417 @@
+:::
+
+## Configure the UID2 Module
+
+You'll need to configure the UID2 Prebid module to complete the following two actions:
+
+| Step | Action | Link to Instructions |
+| --- | --- | --- |
+| 1 | Send a server-side API call to generate a UID2 token.  | [Generating a UID2 Token on the Server](#generating-a-uid2-token-on-the-server) |
+| 2 | Store the response value, so that the Prebid module can manage token refresh as well as opt-out if needed. | [Refreshing a UID2 Token](#refreshing-a-uid2-token) |
+
+### Generating a UID2 Token on the Server
+
+To generate a token, call the [POST /token/generate](../endpoints/post-token-generate.md) endpoint.
+
+For an example, see [Sample Token](#sample-token).
+
+### Refreshing a UID2 Token
+
+There are two ways to refresh a UID2 token, as shown in the following table.
+
+| Mode | Description | Link to Section | 
+| --- | --- | --- |
+| Client refresh mode | Prebid.js automatically refreshes the tokens internally.<br/>This is the simplest approach. | [Client Refresh Mode](#client-refresh-mode) |
+| Server-only mode | Prebid.js doesn't automatically refresh the token. It is up to the publisher to manage token refresh.<br/>Choose this option if you want to use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md) to refresh the token, and Prebid.js to send the token to the bid stream. | [Server-Only Mode](#server-only-mode) |
+
+### Client Refresh Mode
+
+You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+
+- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
+- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+
+For an example, see [Sample Token](#sample-token).
+
+As long as the refresh token remains valid, the UID2 Prebid module refreshes the UID2 token as needed.
+
+This section includes the following information:
+- [Client Refresh Mode Response Storage Options](#client-refresh-mode-response-storage-options)
+- [Client Refresh Mode Cookie Example](#client-refresh-mode-cookie-example)
+- [Client Refresh Mode uid2Token Example](#client-refresh-mode-uid2token-example)
+- [Passing a New Token: Client Refresh Mode](#passing-a-new-token-client-refresh-mode)
+
+#### Client Refresh Mode Response Storage Options
+
+When you configure the module to use Client Refresh mode, you must choose **one** of the following options for storing the API response information.
+
+| Option | Details | Use Case | 
+| --- | --- | --- |
+| Set `params.uid2Cookie` to the name of the cookie that contains the response body as a JSON string. | See [Client Refresh Cookie Example](#client-refresh-cookie-example). | Use this option only when there is enough space left in your cookie to store the response body. (**GWH_LP do we need to address how user will know if there is space in the cookie?**)|
```

Comment:
@lionell-pack-ttd is this optional even for when using local storage (i.e. not using cookie)?

Reply:
@sunnywu : also noted in source doc as an additional query.